test/e2e-wdio/specs/basic.e2e.js

```diff
@@ -23,7 +23,7 @@ describe('Basic flows', () => {
   });
 
   afterEach(async () => {
-    await TestAppPage.ssoLogout();
+    await browser.refresh();
```

Comment:
ssoLogout() step somehow breaks config input in ie11, next test couldn't change new configs
I checked that there no login steps, so decided just to reload page

Reply:
looks like line 95 behaves the login...

For the test app, maybe you can disable the `input` event for ie11, and add a button to set the config at the runtime. The issue is most probably with the test app instead of the logout function.